nextflow

```diff
@@ -15,7 +15,7 @@
 #  limitations under the License.
 
 [[ "$NXF_DEBUG" == 'x' ]] && set -x
-NXF_VER=${NXF_VER:-'19.12.0-edge'}
+NXF_VER=${NXF_VER:-'20.01.0-SNAPSHOT'}
 NXF_ORG=${NXF_ORG:-'nextflow-io'}
```

Comment:
This is generated automatically. SHould not be included in the change set.

Reply:
removed file from commit by
ommit 5a10e40292b8655136e9eca0abe632539af0cd13